android/core/src/main/java/com/pega/vehicledamagemodeling/api/Parser.java

```diff
@@ -35,7 +35,7 @@ public static String parseToMainScreenText(JsonObject jsonObject){
         return text;
     }
 
-    public static HashSet<String> parseToSelectedParts(JsonObject jsonObject){
+    public final HashSet<String> parseToSelectedParts(JsonObject jsonObject){
```

Comment:
So, now you still can't mock Parser, because it has final methods, so you can't override them